packages/babel-plugin-strip-runtime/src/index.tsx

```diff
@@ -0,0 +1,105 @@
+import { declare } from '@babel/helper-plugin-utils';
+import * as t from '@babel/types';
+import {
+  removeStyleDeclarations,
+  isCCComponent,
+  isCreateElement,
+  isAutomaticRuntime,
+} from './utils/ast';
+import type { PluginPass } from './types';
+
+export default declare<PluginPass>((api) => {
+  api.assertVersion(7);
+
+  return {
+    name: '@compiled/babel-plugin-strip-runtime',
+    visitor: {
+      ImportSpecifier(path) {
+        if (t.isIdentifier(path.node.imported) && ['CC', 'CS'].includes(path.node.imported.name)) {
+          path.remove();
+        }
+      },
+      JSXElement(path, pass) {
+        if (!t.isJSXIdentifier(path.node.openingElement.name)) {
+          return;
+        }
+
+        const componentName = path.node.openingElement.name.name;
+        if (componentName !== 'CC') {
+          return;
+        }
+
+        const [, compiledStyles, , nodeToReplace] = path.get('children');
+
+        // Before we replace this node with its children we need to go through and remove all the
+        // style declarations from the CS call.
+        removeStyleDeclarations(compiledStyles.node, path, pass);
+
+        // All done! Let's replace this node with the user land child.
+        path.replaceWith(nodeToReplace);
+        path.node.leadingComments = null;
+        return;
+      },
+      CallExpression(path, pass) {
+        const callee = path.node.callee;
+        if (isCreateElement(callee)) {
+          // We've found something that looks like React.createElement(...)
+          // Now we want to check if it's from the Compiled Runtime and if it is - replace with its children.
+          const component = path.node.arguments[0];
+          if (!isCCComponent(component)) {
+            return;
+          }
+
+          const [, , compiledStyles, nodeToReplace] = path.get('arguments');
```

Comment:
Nit:

`[, ,` this is new to me, I'm assuming we're ignoring the first two params? 

Maybe a prefixing them with a `_` would make this seem a little less weird. 

`[_thing, _otherThing, ....`